### PR TITLE
Update docs after merging repos

### PR DESCRIPTION
**Pull request by joshuali925**
*Issue #, if available:*

*Description of changes:*
- Add notice on readme about merged plugins
- Adjust links in docs for each plugin
- Add instructions for checking out Workbench separately for development

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,17 @@
 
 Open Distro for Elasticsearch enables you to extract insights out of Elasticsearch using the familiar SQL query syntax. Use aggregations, group by, and where clauses to investigate your data. Read your data as JSON documents or CSV tables so you have the flexibility to use the format that works best for you.
 
+
+## SQL Related Plugins
```
Comment (penghuo) [inline README.md:12]: CLI, JDBC, ODBC is not Plugins.

File: docs/dev/Doctest.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ Doctest is relying on the console/command line to run code examples in documenta
     * https://github.com/crate/crate/blob/master/docs/general/dql/selects.rst
 
 
-Similar to CarateDB using it’s CLI “crash”, we can make use of our own [SQL-CLI](https://github.com/opendistro-for-elasticsearch/sql-cli)
+Similar to CarateDB using it’s CLI “crash”, we can make use of our own [SQL-CLI](https://github.com/opendistro-for-elasticsearch/sql/tree/master/sql-cli)
```
Comment (dai-chen) [inline docs/dev/Doctest.md:148]: Probably a typo `CrateDB`? @zhongnansu

Comment (zhongnansu) [reply docs/dev/Doctest.md:148]: Yeah it should be CrateDB https://github.com/crate/crate, please fix it, thx. @joshuali925

Comment (joshuali925) [reply docs/dev/Doctest.md]: Fixed, thanks

File: README.md (hunk 1 above)
Comment (joshuali925) [reply README.md]: Changed "plugins" to "projects"

File: docs/dev/Doctest.md (hunk 2 above)
Comment (dai-chen) [reply docs/dev/Doctest.md]: Thanks zhongnan and Josh!

**Pull request merged**